Tasks/HelmDeployV0/task.json

```diff
@@ -253,7 +253,7 @@
             "name": "overrideValues",
             "label": "Set Values",
             "type": "string",
-            "helpMarkDown": "Set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2). The task will construct the helm command by using these set values. For example, helm install --set key1=val1 ./redis.",
+            "helpMarkDown": "Set values on the command line (can specify multiple or separate values with commas or newlines: key1=val1,key2=val2). The task will construct the helm command by using these set values. For example, helm install --set key1=val1 ./redis.",
```

Comment:
Add examples for newline?

Reply:
Commit the change in `task.loc.json` too

Reply:
The change in strings file (*.resjson) should take care of `task.loc.json` right?

Reply:
Aaah, makes sense.